packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -210,6 +212,14 @@ const ActionListItemBadgeGroup = assignWithoutSideEffects(_ActionListItemBadgeGr
   componentId: componentIds.ActionListItemBadgeGroup,
 });
 
+const _ActionListItemAvatar = (avatarProps: Omit<AvatarProps, 'size' | ''>): React.ReactElement => {
+  return <Avatar size="xsmall" {...avatarProps} />;
```

Comment:
Consumer cannot change size?

Reply:
yup checked with RK. size is always same